pages/public-gateways/faq.mdx

```diff
@@ -53,16 +48,33 @@ The Public Gateways API v1 is now deprecated, and will be removed on 1 October 2
 
 See our [dedicated documentation](/public-gateways/reference-content/understanding-v2/) for full details.
 
-## Do I need a Public Gateway for each Availability Zone (AZ)?
+## Specifications
+
+### Does the Public Gateway require a public IPv4 address?
+
+No. A public IPv4 address (aka. flexible IP) must be assigned to the Public Gateway at creation time, but you can detach it and delete it afterward if you do not need it to be publicly accessible.
+
+### Do I need a Public Gateway for each Availability Zone (AZ)?
 
 VPC and Private Networks are both [regional](/public-gateways/concepts/#region-and-availability-zone), meaning they span all AZs across a given region. Even though Public Gateways are zoned and not regional, one Public Gateway attached to a regional Private Network is functionally enough, and will cover the whole region. That is to say a Public Gateway created in PAR-1 can serve Instances in PAR-2 and PAR-3, as long as they are all attached to the same PAR-region Private Network.
 
-## How can I achieve truly High Availability (HA) networking when the Public Gateway is a zoned resource? 
+## Access and security
+
+### Can my Instances and other resources access the internet via a Public Gateway without a public IP address?
+
+Yes. The Public Gateway can advertise itself as the [default route to the internet](/public-gateways/concepts/#default-route) over the Private Network it is attached to, so that Instances and other resources can access the internet via the gateway. Resources attached to other Private Networks than the gateway's network in the VPC can [opt in](/vpc/how-to/manage-routing/#how-to-manage-default-route-scope) to receive its default route advertisement.
+Moreover, the Public Gateway supports [static NAT](/public-gateways/how-to/configure-a-public-gateway/#how-to-review-and-configure-nat) (aka. port forwarding), so that ingress traffic from the public internet can reach Instances on the Private Network. This works by mapping pre-defined ports of the public IP address of the gateway to specific ports and IP addresses on the VPC.
```

Comment:
```suggestion
Moreover, the Public Gateway supports [static NAT](/public-gateways/how-to/configure-a-public-gateway/#how-to-review-and-configure-nat) (also known as port forwarding), so that ingress traffic from the public internet can reach Instances on the Private Network. This works by mapping pre-defined ports of the public IP address of the gateway to specific ports and IP addresses on the VPC.
```